code/modules/mob/new_player/new_player.dm

```diff
@@ -246,6 +246,8 @@
 	if(!is_job_whitelisted(src, rank))	 return 0
 	if(!job.player_old_enough(src.client))	return 0
 	if(job.admin_only && !(check_rights(R_EVENT, 0))) return 0
+	if(job.available_in_playtime(src.client))

```

Comment:
You don't need to use `src.client` here. Just use `client`. And yes, the line two up from this also uses an unnecessary `src.` - don't repeat their mistake.